--- src/jobs/meltjob.cpp.orig	2020-11-02 15:30:43.463144678 +0100
+++ src/jobs/meltjob.cpp	2020-11-02 15:30:59.196142990 +0100
@@ -95,7 +95,7 @@
         return;
     }
     QString shotcutPath = qApp->applicationDirPath();
-    QFileInfo meltPath(shotcutPath, "melt");
+    QFileInfo meltPath(shotcutPath, "mlt-melt");
     setReadChannel(QProcess::StandardError);
     QStringList args;
     args << "-verbose";
