--- src/jobs/meltjob.cpp.orig	2021-10-30 23:37:09.000000000 +0100
+++ src/jobs/meltjob.cpp	2021-12-12 10:17:16.258891583 +0000
@@ -96,7 +96,7 @@ void MeltJob::start()
     }
     QString shotcutPath = qApp->applicationDirPath();
 #if defined(Q_OS_UNIX) && !defined(Q_OS_MAC)
-    QFileInfo meltPath(shotcutPath, "melt-7");
+    QFileInfo meltPath(shotcutPath, "mlt-melt");
 #else
     QFileInfo meltPath(shotcutPath, "melt");
 #endif
