--- src/mainwindow.cpp.orig	2018-10-02 13:48:27.170388091 +0200
+++ src/mainwindow.cpp	2018-10-02 13:49:59.172328038 +0200
@@ -2790,9 +2790,9 @@
         showStatusMessage("Checking for upgrade...");
         m_network.get(QNetworkRequest(QUrl("http://check.shotcut.org/version.json")));
     } else {
-        QAction* action = new QAction(tr("Click here to check for a new version of Shotcut."), 0);
-        connect(action, SIGNAL(triggered(bool)), SLOT(on_actionUpgrade_triggered()));
-        showStatusMessage(action, 15 /* seconds */);
+        //QAction* action = new QAction(tr("Click here to check for a new version of Shotcut."), 0);
+        //connect(action, SIGNAL(triggered(bool)), SLOT(on_actionUpgrade_triggered()));
+        //showStatusMessage(action, 15 /* seconds */);
     }
 }
 
@@ -3375,7 +3375,7 @@
                     m_upgradeUrl = json.object().value("url").toString();
                 showStatusMessage(action, 15 /* seconds */);
             } else {
-                showStatusMessage(tr("You are running the latest version of Shotcut."));
+		 showStatusMessage(tr("Use dnf for updates."));
             }
             reply->deleteLater();
             return;
--- src/src.pro.orig	2018-10-02 13:52:58.638721580 +0200
+++ src/src.pro	2018-10-02 13:53:27.064761829 +0200
@@ -300,7 +300,7 @@
 LIBS += -lCuteLogger
 
 isEmpty(SHOTCUT_VERSION) {
-    !win32:SHOTCUT_VERSION = $$system(date "+%y.%m.%d")
+    !win32:SHOTCUT_VERSION = $$_VSTRING
      win32:SHOTCUT_VERSION = adhoc
 }
 DEFINES += SHOTCUT_VERSION=\\\"$$SHOTCUT_VERSION\\\"
