Index: src/mainwindow.cpp
===================================================================
--- src/mainwindow.cpp.orig	2017-12-08 13:38:56.944162254 +0100
+++ src/mainwindow.cpp	2017-12-08 13:42:07.385176045 +0100
@@ -2548,9 +2548,9 @@
 
 void MainWindow::showUpgradePrompt()
 {
-    QAction* action = new QAction(tr("Click here to check for a new version of Shotcut."), 0);
-    connect(action, SIGNAL(triggered(bool)), SLOT(on_actionUpgrade_triggered()));
-    showStatusMessage(action, 15 /* seconds */);
+    //QAction* action = new QAction(tr("Click here to check for a new version of Shotcut."), 0);
+    //connect(action, SIGNAL(triggered(bool)), SLOT(on_actionUpgrade_triggered()));
+    //showStatusMessage(action, 15 /* seconds */);
 }
 
 void MainWindow::on_actionRealtime_triggered(bool checked)
@@ -2909,7 +2909,6 @@
 void MainWindow::on_actionUpgrade_triggered()
 {
     showStatusMessage("Checking for upgrade...");
-    m_network.get(QNetworkRequest(QUrl("http://check.shotcut.org/version.json")));
 }
 
 void MainWindow::on_actionOpenXML_triggered()
@@ -3041,7 +3040,7 @@
                     m_upgradeUrl = json.object().value("url").toString();
                 showStatusMessage(action, 10 /* seconds */);
             } else {
-                showStatusMessage(tr("You are running the latest version of Shotcut."));
+                showStatusMessage(tr("Use dnf for updates."));
             }
             reply->deleteLater();
             return;
Index: src/src.pro
===================================================================
--- src/src.pro.orig	2017-12-08 13:42:45.581178811 +0100
+++ src/src.pro	2017-12-08 13:44:10.984184995 +0100
@@ -329,7 +329,7 @@
 LIBS += -lCuteLogger -lmvcp -lpthread
 
 isEmpty(SHOTCUT_VERSION) {
-    !win32:SHOTCUT_VERSION = $$system(date "+%y.%m.%d")
+    !win32:SHOTCUT_VERSION = $$_VSTRING
      win32:SHOTCUT_VERSION = adhoc
 }
 DEFINES += SHOTCUT_VERSION=\\\"$$SHOTCUT_VERSION\\\"
