--- src/mainwindow.cpp.orig	2020-09-18 10:39:00.943983974 +0200
+++ src/mainwindow.cpp	2020-09-18 10:42:54.653011748 +0200
@@ -3302,9 +3302,9 @@
         showStatusMessage("Checking for upgrade...");
         m_network.get(QNetworkRequest(QUrl("https://check.shotcut.org/version.json")));
     } else {
-        QAction* action = new QAction(tr("Click here to check for a new version of Shotcut."), 0);
-        connect(action, SIGNAL(triggered(bool)), SLOT(on_actionUpgrade_triggered()));
-        showStatusMessage(action, 15 /* seconds */);
+        //QAction* action = new QAction(tr("Click here to check for a new version of Shotcut."), 0);
+        //connect(action, SIGNAL(triggered(bool)), SLOT(on_actionUpgrade_triggered()));
+        //showStatusMessage(action, 15 /* seconds */);
     }
 }
 
@@ -3937,7 +3937,7 @@
                     m_upgradeUrl = json.object().value("url").toString();
                 showStatusMessage(action, 15 /* seconds */);
             } else {
-                showStatusMessage(tr("You are running the latest version of Shotcut."));
+                showStatusMessage(tr("Use dnf for updates."));
             }
             reply->deleteLater();
             return;
